app/assets/stylesheets/main.scss

```diff
@@ -66,7 +66,25 @@ textarea#auction_description {
 }
 
 .github-issue {
-  white-space: pre-line;
+  h1 {
+	  line-height: 1.1em;
+	  margin-top: 0.2em;
+	  margin-bottom: 0.5em;
+  }
+
+  h2, h3, h4, h5, h6 {
+	  line-height: 1.1em;
+	  margin-top: 0.2em;
+	  margin-bottom: 0.1em;
+  }
+
+  p, li {
+	  line-height: 1.3em;
+  }
+
+  ul {
+	  margin-bottom: 0.5em;
+  }
 }
```

Comment:
I found when just removing `white-space: pre-line`, the vertical spacing was still a bit larger than the 30px max desired.

![2016-01-14-074916_1029x1313_scrot](https://cloud.githubusercontent.com/assets/387035/12329092/7e0e2212-ba93-11e5-975b-7c8d18f63b40.png)